CSharpSourceCode/AttributeDataSystem/PartyAttribute.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.SaveSystem;
+
+namespace TOW_Core.AttributeDataSystem
+{
+    /// <summary>
+    /// Contains TOW troop data of a complete mobile Party. For single entity data use a StaticAttribute
+    /// </summary>
+    public class PartyAttribute
+    {
+        [SaveableField(1)]
+        public Hero Leader;
+        [SaveableField(2)]
+        public string id;           //partybase.
+        [SaveableField(3)]
+        public float WindsOfMagic;
+        [SaveableField(4)]
+        public bool IsMagicUserParty;
+        [SaveableField(5)]
+        public StaticAttribute LeaderAttribute;
+        [SaveableField(6)]
+        public List<StaticAttribute> CompanionAttributes = new List<StaticAttribute>();
+        [SaveableField(7)]
+        public List<StaticAttribute> RegularTroopAttributes = new List<StaticAttribute>();
+
+        [SaveableField(8)] public PartyType PartyType;
+        [SaveableField(9)] public int numberOfRegularTroops;
```

Comment:
Capital N `NumberOfRegularTroops` to maintain a consistent convention for these public fields?

Also, for lines 27, 28, and 29, is there a reason the `SaveableField` attribute is on the same line as the field? I personally think it's better to have it above the field but I'd like it to at least be consistent.

Reply:
Was removed in recent version